diff -up blender-3.2.2/source/blender/blenfont/intern/blf_glyph.c.5~ blender-3.2.2/source/blender/blenfont/intern/blf_glyph.c
--- blender-3.2.2/source/blender/blenfont/intern/blf_glyph.c.5~	2022-08-11 02:07:38.241277996 +0200
+++ blender-3.2.2/source/blender/blenfont/intern/blf_glyph.c	2022-08-11 02:27:05.812113176 +0200
@@ -272,7 +272,7 @@ static FT_GlyphSlot blf_glyph_load(FontB
  */
 static bool blf_glyph_render_bitmap(FontBLF *font, FT_GlyphSlot glyph)
 {
-  int render_mode;
+  unsigned int render_mode;
 
   if (font->flags & BLF_MONOCHROME) {
     render_mode = FT_RENDER_MODE_MONO;
diff -up blender-3.2.2/source/blender/blenkernel/intern/idprop_utils.c.5~ blender-3.2.2/source/blender/blenkernel/intern/idprop_utils.c
--- blender-3.2.2/source/blender/blenkernel/intern/idprop_utils.c.5~	2022-05-02 15:58:51.000000000 +0200
+++ blender-3.2.2/source/blender/blenkernel/intern/idprop_utils.c	2022-08-11 02:07:38.241277996 +0200
@@ -170,7 +170,7 @@ static void idp_repr_fn_recursive(struct
       const ID *id = prop->data.pointer;
       if (id != NULL) {
         STR_APPEND_STR("bpy.data.");
-        STR_APPEND_STR(BKE_idtype_idcode_to_name_plural(GS(id->name)));
+        STR_APPEND_STR(BKE_idtype_idcode_to_name_plural((short)GS(id->name)));
         STR_APPEND_STR("[");
         STR_APPEND_STR_QUOTE(id->name + 2);
         STR_APPEND_STR("]");
diff -up blender-3.2.2/source/blender/blenkernel/intern/mball_tessellate.c.5~ blender-3.2.2/source/blender/blenkernel/intern/mball_tessellate.c
--- blender-3.2.2/source/blender/blenkernel/intern/mball_tessellate.c.5~	2022-05-02 15:58:52.000000000 +0200
+++ blender-3.2.2/source/blender/blenkernel/intern/mball_tessellate.c	2022-08-11 02:07:38.241277996 +0200
@@ -36,6 +36,9 @@
 
 #include "BLI_strict_flags.h"
 
+#pragma GCC diagnostic ignored "-Wsign-conversion"
+#pragma clang diagnostic ignored "-Wsign-conversion"
+
 /* experimental (faster) normal calculation */
 // #define USE_ACCUM_NORMAL
 
diff -up blender-3.2.2/source/blender/blenkernel/intern/mesh_remap.c.5~ blender-3.2.2/source/blender/blenkernel/intern/mesh_remap.c
--- blender-3.2.2/source/blender/blenkernel/intern/mesh_remap.c.5~	2022-05-02 15:58:52.000000000 +0200
+++ blender-3.2.2/source/blender/blenkernel/intern/mesh_remap.c	2022-08-11 02:07:38.242277971 +0200
@@ -33,6 +33,9 @@
 
 #include "BLI_strict_flags.h"
 
+#pragma GCC diagnostic ignored "-Wfloat-conversion"
+#pragma clang diagnostic ignored "-Wfloat-conversion"
+
 static CLG_LogRef LOG = {"bke.mesh"};
 
 /* -------------------------------------------------------------------- */
diff -up blender-3.2.2/source/blender/blenlib/intern/easing.c.5~ blender-3.2.2/source/blender/blenlib/intern/easing.c
--- blender-3.2.2/source/blender/blenlib/intern/easing.c.5~	2022-05-02 15:58:53.000000000 +0200
+++ blender-3.2.2/source/blender/blenlib/intern/easing.c	2022-08-11 02:07:38.242277971 +0200
@@ -11,6 +11,9 @@
 
 #include "BLI_strict_flags.h"
 
+#pragma GCC diagnostic ignored "-Wfloat-conversion"
+#pragma clang diagnostic ignored "-Wfloat-conversion"
+
 /* blend if (amplitude < fabsf(change) */
 #define USE_ELASTIC_BLEND
 
diff -up blender-3.2.2/source/blender/blenlib/intern/math_geom.c.5~ blender-3.2.2/source/blender/blenlib/intern/math_geom.c
--- blender-3.2.2/source/blender/blenlib/intern/math_geom.c.5~	2022-05-02 15:58:53.000000000 +0200
+++ blender-3.2.2/source/blender/blenlib/intern/math_geom.c	2022-08-11 02:07:38.243277947 +0200
@@ -13,6 +13,9 @@
 
 #include "BLI_strict_flags.h"
 
+#pragma GCC diagnostic ignored "-Wfloat-conversion"
+#pragma clang diagnostic ignored "-Wfloat-conversion"
+
 /********************************** Polygons *********************************/
 
 void cross_tri_v3(float n[3], const float v1[3], const float v2[3], const float v3[3])
diff -up blender-3.2.2/source/blender/blenlib/intern/math_rotation.c.5~ blender-3.2.2/source/blender/blenlib/intern/math_rotation.c
--- blender-3.2.2/source/blender/blenlib/intern/math_rotation.c.5~	2022-05-02 15:58:53.000000000 +0200
+++ blender-3.2.2/source/blender/blenlib/intern/math_rotation.c	2022-08-11 02:07:38.243277947 +0200
@@ -9,6 +9,9 @@
 
 #include "BLI_strict_flags.h"
 
+#pragma GCC diagnostic ignored "-Wfloat-conversion"
+#pragma clang diagnostic ignored "-Wfloat-conversion"
+
 /******************************** Quaternions ********************************/
 
 /* used to test is a quat is not normalized (only used for debug prints) */
diff -up blender-3.2.2/source/blender/modifiers/intern/MOD_correctivesmooth.c.5~ blender-3.2.2/source/blender/modifiers/intern/MOD_correctivesmooth.c
--- blender-3.2.2/source/blender/modifiers/intern/MOD_correctivesmooth.c.5~	2022-08-11 02:07:38.243277947 +0200
+++ blender-3.2.2/source/blender/modifiers/intern/MOD_correctivesmooth.c	2022-08-11 02:28:06.680572624 +0200
@@ -197,7 +197,7 @@ static void smooth_iter__simple(Correcti
    * here to avoid multiplying for every iteration */
   if (smooth_weights == NULL) {
     for (i = 0; i < verts_num; i++) {
-      vertex_edge_count_div[i] = lambda * (vertex_edge_count_div[i] ?
+      vertex_edge_count_div[i] = lambda * ((vertex_edge_count_div[i] != 0.0f) ?
                                                (1.0f / vertex_edge_count_div[i]) :
                                                1.0f);
     }
@@ -205,7 +205,7 @@ static void smooth_iter__simple(Correcti
   else {
     for (i = 0; i < verts_num; i++) {
       vertex_edge_count_div[i] = smooth_weights[i] * lambda *
-                                 (vertex_edge_count_div[i] ? (1.0f / vertex_edge_count_div[i]) :
+                                 ((vertex_edge_count_div[i] != 0.0f) ? (1.0f / vertex_edge_count_div[i]) :
                                                              1.0f);
     }
   }
diff -up blender-3.2.2/source/blender/modifiers/intern/MOD_screw.c.5~ blender-3.2.2/source/blender/modifiers/intern/MOD_screw.c
--- blender-3.2.2/source/blender/modifiers/intern/MOD_screw.c.5~	2022-05-12 15:29:09.000000000 +0200
+++ blender-3.2.2/source/blender/modifiers/intern/MOD_screw.c	2022-08-11 02:07:38.244277923 +0200
@@ -41,6 +41,9 @@
 #include "MOD_modifiertypes.h"
 #include "MOD_ui_common.h"
 
+#pragma GCC diagnostic ignored "-Wfloat-conversion"
+#pragma clang diagnostic ignored "-Wfloat-conversion"
+
 static void initData(ModifierData *md)
 {
   ScrewModifierData *ltmd = (ScrewModifierData *)md;
@@ -418,7 +421,7 @@ static Mesh *modifyMesh(ModifierData *md
     }
 
     uv_v_range_inv = uv_v_minmax[1] - uv_v_minmax[0];
-    uv_v_range_inv = uv_v_range_inv ? 1.0f / uv_v_range_inv : 0.0f;
+    uv_v_range_inv = (uv_v_range_inv != 0.0f) ? 1.0f / uv_v_range_inv : 0.0f;
   }
 
   /* Set the locations of the first set of verts */
@@ -859,7 +862,7 @@ static Mesh *modifyMesh(ModifierData *md
     }
     copy_m4_m3(mat, mat3);
 
-    if (screw_ofs) {
+    if (screw_ofs != 0.0f) {
       madd_v3_v3fl(mat[3], axis_vec, screw_ofs * ((float)step / (float)(step_tot - 1)));
     }
 
