diff -p -up blender-2.65a/source/blender/blenfont/intern/blf_lang.c.localedir blender-2.65a/source/blender/blenfont/intern/blf_lang.c
--- blender-2.65a/source/blender/blenfont/intern/blf_lang.c.localedir	2012-11-11 17:54:26.000000000 +0100
+++ blender-2.65a/source/blender/blenfont/intern/blf_lang.c	2012-12-21 11:52:28.548683204 +0100
@@ -83,15 +83,13 @@ static void free_locales(void)
 
 static void fill_locales(void)
 {
-	char *languages_path = BLI_get_folder(BLENDER_DATAFILES, "locale");
 	LinkNode *lines = NULL, *line;
 	char *str;
 	int idx = 0;
 
 	free_locales();
 
-	BLI_join_dirfile(languages_path, FILE_MAX, languages_path, "languages");
-	line = lines = BLI_file_read_as_lines(languages_path);
+	line = lines = BLI_file_read_as_lines("/usr/share/blender/locale/languages");
 
 	/* This whole "parsing" code is a bit weak, in that it expects strictly formated input file...
 	 * Should not be a problem, though, as this file is script-generated! */
@@ -184,7 +182,7 @@ EnumPropertyItem *BLF_RNA_lang_enum_prop
 
 void BLF_lang_init(void)
 {
-	char *messagepath = BLI_get_folder(BLENDER_DATAFILES, "locale");
+	char *messagepath = "/usr/share/locale";
 
 	if (messagepath) {
 		bl_locale_init(messagepath, TEXT_DOMAIN_NAME);
diff -p -up blender-2.65a/source/creator/CMakeLists.txt.localedir blender-2.65a/source/creator/CMakeLists.txt
--- blender-2.65a/source/creator/CMakeLists.txt.localedir	2012-11-26 13:38:40.000000000 +0100
+++ blender-2.65a/source/creator/CMakeLists.txt	2012-12-21 11:50:27.305900176 +0100
@@ -295,6 +295,16 @@ if(WITH_INTERNATIONAL)
 	install(
 		DIRECTORY
 			${CMAKE_SOURCE_DIR}/release/datafiles/locale
+		DESTINATION share
+		PATTERN ".svn" EXCLUDE
+		PATTERN "languages" EXCLUDE
+	)
+	install(FILES
+			${CMAKE_SOURCE_DIR}/release/datafiles/locale/languages
+		DESTINATION share/blender/locale
+	)
+	install(
+		DIRECTORY
 			${CMAKE_SOURCE_DIR}/release/datafiles/fonts
 		DESTINATION ${TARGETDIR_VER}/datafiles
 		PATTERN ".svn" EXCLUDE
