diff -up blender-2.80/blender.thumbnailer.6~ blender-2.80/blender.thumbnailer
--- blender-2.80/blender.thumbnailer.6~	2019-07-31 15:30:00.571305028 +0200
+++ blender-2.80/blender.thumbnailer	2019-07-31 15:30:00.570305046 +0200
@@ -0,0 +1,4 @@
+[Thumbnailer Entry]
+TryExec=/usr/share/blender/scripts/blender-thumbnailer.py
+Exec=/usr/share/blender/scripts/blender-thumbnailer.py %i %o
+MimeType=application/x-blender;
diff -up blender-2.80/source/creator/CMakeLists.txt.6~ blender-2.80/source/creator/CMakeLists.txt
--- blender-2.80/source/creator/CMakeLists.txt.6~	2019-07-31 15:30:00.571305028 +0200
+++ blender-2.80/source/creator/CMakeLists.txt	2019-07-31 15:34:17.306097169 +0200
@@ -510,7 +510,7 @@ if(UNIX AND NOT APPLE)
     install(
       PROGRAMS
       ${CMAKE_SOURCE_DIR}/release/bin/blender-thumbnailer.py
-      DESTINATION "."
+      DESTINATION ${CMAKE_INSTALL_PREFIX}/share/blender/scripts
     )
 
     set(BLENDER_TEXT_FILES_DESTINATION ".")
