diff -up blender-2.80/source/blender/imbuf/intern/jp2.c.openjpeg blender-2.80/source/blender/imbuf/intern/jp2.c
--- blender-2.80/source/blender/imbuf/intern/jp2.c.openjpeg	2019-07-31 15:23:07.765846849 +0200
+++ blender-2.80/source/blender/imbuf/intern/jp2.c	2019-07-31 15:23:36.359687452 +0200
@@ -30,6 +30,7 @@
 #include "IMB_colormanagement.h"
 #include "IMB_colormanagement_intern.h"
 
+#include <stdbool.h>
 #include "openjpeg.h"
 
 #define JP2_FILEHEADER_SIZE 12
