diff -urN blender-2.64/intern/cycles/bvh/CMakeLists.txt blender-2.64-patched/intern/cycles/bvh/CMakeLists.txt
--- blender-2.64/intern/cycles/bvh/CMakeLists.txt	2012-10-04 01:34:36.000000000 +1100
+++ blender-2.64-patched/intern/cycles/bvh/CMakeLists.txt	2012-10-06 12:18:30.746376681 +1100
@@ -32,4 +32,4 @@
 include_directories(${INC})
 include_directories(SYSTEM ${INC_SYS})
 
-add_library(cycles_bvh ${SRC} ${SRC_HEADERS})
+add_library(cycles_bvh STATIC ${SRC} ${SRC_HEADERS})
diff -urN blender-2.64/intern/cycles/device/CMakeLists.txt blender-2.64-patched/intern/cycles/device/CMakeLists.txt
--- blender-2.64/intern/cycles/device/CMakeLists.txt	2012-10-04 01:34:36.000000000 +1100
+++ blender-2.64-patched/intern/cycles/device/CMakeLists.txt	2012-10-06 12:18:24.625376966 +1100
@@ -35,4 +35,4 @@
 include_directories(${INC})
 include_directories(SYSTEM ${INC_SYS})
 
-add_library(cycles_device ${SRC} ${SRC_HEADERS})
+add_library(cycles_device STATIC ${SRC} ${SRC_HEADERS})
diff -urN blender-2.64/intern/cycles/kernel/CMakeLists.txt blender-2.64-patched/intern/cycles/kernel/CMakeLists.txt
--- blender-2.64/intern/cycles/kernel/CMakeLists.txt	2012-10-04 01:35:35.000000000 +1100
+++ blender-2.64-patched/intern/cycles/kernel/CMakeLists.txt	2012-10-06 12:18:12.091377739 +1100
@@ -139,7 +139,7 @@
 include_directories(${INC})
 include_directories(SYSTEM ${INC_SYS})
 
-add_library(cycles_kernel ${SRC} ${SRC_HEADERS} ${SRC_SVM_HEADERS})
+add_library(cycles_kernel STATIC ${SRC} ${SRC_HEADERS} ${SRC_SVM_HEADERS})
 
 if(WITH_CYCLES_OPTIMIZED_KERNEL)
 	set_source_files_properties(kernel_optimized.cpp PROPERTIES COMPILE_FLAGS "${CYCLES_OPTIMIZED_KERNEL_FLAGS}")
diff -urN blender-2.64/intern/cycles/render/CMakeLists.txt blender-2.64-patched/intern/cycles/render/CMakeLists.txt
--- blender-2.64/intern/cycles/render/CMakeLists.txt	2012-10-04 01:35:28.000000000 +1100
+++ blender-2.64-patched/intern/cycles/render/CMakeLists.txt	2012-10-06 12:17:43.197379576 +1100
@@ -68,5 +68,5 @@
 include_directories(${INC})
 include_directories(SYSTEM ${INC_SYS})
 
-add_library(cycles_render ${SRC} ${SRC_HEADERS})
+add_library(cycles_render STATIC ${SRC} ${SRC_HEADERS})
 
diff -urN blender-2.64/intern/cycles/subd/CMakeLists.txt blender-2.64-patched/intern/cycles/subd/CMakeLists.txt
--- blender-2.64/intern/cycles/subd/CMakeLists.txt	2012-10-04 01:35:37.000000000 +1100
+++ blender-2.64-patched/intern/cycles/subd/CMakeLists.txt	2012-10-06 12:17:36.917379604 +1100
@@ -35,4 +35,4 @@
 include_directories(${INC})
 include_directories(SYSTEM ${INC_SYS})
 
-add_library(cycles_subd ${SRC} ${SRC_HEADERS})
+add_library(cycles_subd STATIC ${SRC} ${SRC_HEADERS})
diff -urN blender-2.64/intern/cycles/util/CMakeLists.txt blender-2.64-patched/intern/cycles/util/CMakeLists.txt
--- blender-2.64/intern/cycles/util/CMakeLists.txt	2012-10-04 01:35:40.000000000 +1100
+++ blender-2.64-patched/intern/cycles/util/CMakeLists.txt	2012-10-06 12:17:21.222380712 +1100
@@ -68,4 +68,4 @@
 include_directories(${INC})
 include_directories(SYSTEM ${INC_SYS})
 
-add_library(cycles_util ${SRC} ${SRC_HEADERS})
+add_library(cycles_util STATIC ${SRC} ${SRC_HEADERS})
