diff -up blender-4.0.0/source/blender/io/alembic/CMakeLists.txt.1~ blender-4.0.0/source/blender/io/alembic/CMakeLists.txt
--- blender-4.0.0/source/blender/io/alembic/CMakeLists.txt.1~	2023-11-14 14:44:35.165591093 +0100
+++ blender-4.0.0/source/blender/io/alembic/CMakeLists.txt	2023-11-14 14:45:47.669504994 +0100
@@ -104,7 +104,7 @@ if(WITH_BOOST)
   )
 endif()
 
-blender_add_lib(bf_io_alembic "${SRC}" "${INC}" "${INC_SYS}" "${LIB}")
+blender_add_lib(bf_io_alembic "${SRC}" "${INC}" "${INC_SYS}" "${LIB}" atomic)
 
 if(WITH_GTESTS)
   set(TEST_SRC
diff -up blender-4.0.0/source/creator/CMakeLists.txt.1~ blender-4.0.0/source/creator/CMakeLists.txt
--- blender-4.0.0/source/creator/CMakeLists.txt.1~	2023-11-14 14:44:35.165591093 +0100
+++ blender-4.0.0/source/creator/CMakeLists.txt	2023-11-14 14:45:27.778250942 +0100
@@ -1598,7 +1598,7 @@ endif()
 # Setup link libraries
 
 add_dependencies(blender makesdna)
-target_link_libraries(blender PRIVATE ${LIB})
+target_link_libraries(blender PRIVATE ${LIB} atomic)
 unset(LIB)
 
 setup_platform_linker_flags(blender)
