--- blender-v2.70a/intern/cycles/render/CMakeLists.txt.orig	2014-04-11 15:46:41.000000000 +0100
+++ blender-v2.70a/intern/cycles/render/CMakeLists.txt	2014-04-20 01:52:27.302437255 +0100
@@ -74,4 +74,4 @@

 add_definitions(${GL_DEFINITIONS})
 
-add_library(cycles_render ${SRC} ${SRC_HEADERS})
+add_library(cycles_render STATIC ${SRC} ${SRC_HEADERS})
--- blender-v2.70a/intern/cycles/bvh/CMakeLists.txt.orig	2014-04-11 15:46:41.000000000 +0100
+++ blender-v2.70a/intern/cycles/bvh/CMakeLists.txt	2014-04-20 01:52:27.300437255 +0100
@@ -33,4 +33,4 @@
 include_directories(${INC})
 include_directories(SYSTEM ${INC_SYS})
 
-add_library(cycles_bvh ${SRC} ${SRC_HEADERS})
+add_library(cycles_bvh STATIC ${SRC} ${SRC_HEADERS})
--- blender-v2.70a/intern/cycles/util/CMakeLists.txt.orig	2014-04-11 15:46:41.000000000 +0100
+++ blender-v2.70a/intern/cycles/util/CMakeLists.txt	2014-04-20 01:52:27.302437255 +0100
@@ -69,4 +69,4 @@

 add_definitions(${GL_DEFINITIONS})

-add_library(cycles_util ${SRC} ${SRC_HEADERS})
+add_library(cycles_util STATIC ${SRC} ${SRC_HEADERS})
--- blender-v2.70a/intern/cycles/subd/CMakeLists.txt.orig	2014-04-11 15:46:41.000000000 +0100
+++ blender-v2.70a/intern/cycles/subd/CMakeLists.txt	2014-04-20 01:52:27.302437255 +0100
@@ -28,4 +28,4 @@
 include_directories(${INC})
 include_directories(SYSTEM ${INC_SYS})
 
-add_library(cycles_subd ${SRC} ${SRC_HEADERS})
+add_library(cycles_subd STATIC ${SRC} ${SRC_HEADERS})
--- blender-v2.70a/intern/cycles/device/CMakeLists.txt.orig	2014-04-11 15:46:41.000000000 +0100
+++ blender-v2.70a/intern/cycles/device/CMakeLists.txt	2014-04-20 01:52:27.301437255 +0100
@@ -41,4 +41,4 @@
 include_directories(${INC})
 include_directories(SYSTEM ${INC_SYS})
 
-add_library(cycles_device ${SRC} ${SRC_HEADERS})
+add_library(cycles_device STATIC ${SRC} ${SRC_HEADERS})
--- blender-v2.70a/intern/cycles/kernel/osl/CMakeLists.txt.orig	2014-04-11 15:46:41.000000000 +0100
+++ blender-v2.70a/intern/cycles/kernel/osl/CMakeLists.txt	2014-04-20 01:52:27.301437255 +0100
@@ -36,5 +36,5 @@
 include_directories(${INC})
 include_directories(SYSTEM ${INC_SYS})
 
-add_library(cycles_kernel_osl ${SRC} ${HEADER_SRC})
+add_library(cycles_kernel_osl STATIC ${SRC} ${HEADER_SRC})
 
--- blender-v2.70a/intern/cycles/kernel/CMakeLists.txt.orig	2014-04-11 15:50:37.000000000 +0100
+++ blender-v2.70a/intern/cycles/kernel/CMakeLists.txt	2014-04-20 01:52:27.301437255 +0100
@@ -238,7 +238,7 @@
 	set_source_files_properties(kernel_avx2.cpp PROPERTIES COMPILE_FLAGS "${CYCLES_AVX2_KERNEL_FLAGS}")
 endif()

-add_library(cycles_kernel ${SRC} ${SRC_HEADERS} ${SRC_CLOSURE_HEADERS} ${SRC_SVM_HEADERS} ${SRC_GEOM_HEADERS})
+add_library(cycles_kernel STATIC ${SRC} ${SRC_HEADERS} ${SRC_CLOSURE_HEADERS} ${SRC_SVM_HEADERS} ${SRC_GEOM_HEADERS})
 
 if(WITH_CYCLES_CUDA)
 	add_dependencies(cycles_kernel cycles_kernel_cuda)
