diff -up blender-2.80/source/creator/CMakeLists.txt.linkage~ blender-2.80/source/creator/CMakeLists.txt
--- blender-2.80/source/creator/CMakeLists.txt.linkage~	2019-07-31 15:23:57.579568933 +0200
+++ blender-2.80/source/creator/CMakeLists.txt	2019-07-31 15:25:07.906174856 +0200
@@ -1045,7 +1045,7 @@ install(
 # Setup link libs
 
 add_dependencies(blender makesdna)
-target_link_libraries(blender ${LIB})
+target_link_libraries(blender ${LIB} icuuc icui18n)
 unset(LIB)
 
 setup_liblinks(blender)
