diff -Naur blender-2.67.orig/source/blender/blenfont/intern/blf_lang.c blender-2.67/source/blender/blenfont/intern/blf_lang.c
--- blender-2.67.orig/source/blender/blenfont/intern/blf_lang.c	2013-05-08 18:00:00.984249791 +0400
+++ blender-2.67/source/blender/blenfont/intern/blf_lang.c	2013-05-08 18:05:10.139541364 +0400
@@ -78,7 +78,6 @@
 
 static void fill_locales(void)
 {
-	const char * const languages_path = BLI_get_folder(BLENDER_DATAFILES, "locale");
 	char languages[FILE_MAX];
 	LinkNode *lines = NULL, *line;
 	char *str;
@@ -86,8 +85,7 @@
 
 	free_locales();
 
-	BLI_join_dirfile(languages, FILE_MAX, languages_path, "languages");
-	line = lines = BLI_file_read_as_lines(languages);
+	line = lines = BLI_file_read_as_lines("/usr/share/blender/locale/languages");
 
 	/* This whole "parsing" code is a bit weak, in that it expects strictly formated input file...
 	 * Should not be a problem, though, as this file is script-generated! */
@@ -186,7 +184,7 @@
 void BLF_lang_init(void)
 {
 #ifdef WITH_INTERNATIONAL
-	const char * const messagepath = BLI_get_folder(BLENDER_DATAFILES, "locale");
+	const char * const messagepath = "/usr/share/locale";
 
 	if (messagepath) {
 		bl_locale_init(messagepath, TEXT_DOMAIN_NAME);
diff -Naur blender-2.67.orig/source/creator/CMakeLists.txt blender-2.67/source/creator/CMakeLists.txt
--- blender-2.67.orig/source/creator/CMakeLists.txt	2013-05-08 17:59:57.849247177 +0400
+++ blender-2.67/source/creator/CMakeLists.txt	2013-05-08 18:02:33.042394763 +0400
@@ -309,6 +309,16 @@
 	install(
 		DIRECTORY
 			${CMAKE_SOURCE_DIR}/release/datafiles/locale
+		DESTINATION share
+		PATTERN ".svn" EXCLUDE
+		PATTERN "languages" EXCLUDE
+	)
+	install(FILES
+			${CMAKE_SOURCE_DIR}/release/datafiles/locale/languages
+		DESTINATION share/blender/locale
+	)
+	install(
+		DIRECTORY
 			${CMAKE_SOURCE_DIR}/release/datafiles/fonts
 		DESTINATION ${TARGETDIR_VER}/datafiles
 		PATTERN ".svn" EXCLUDE
