diff -up blender-3.6.0/source/creator/CMakeLists.txt.2~ blender-3.6.0/source/creator/CMakeLists.txt
--- blender-3.6.0/source/creator/CMakeLists.txt.2~	2023-06-28 19:06:46.921894081 +0200
+++ blender-3.6.0/source/creator/CMakeLists.txt	2023-06-28 19:12:36.992177424 +0200
@@ -438,7 +438,7 @@ if(WITH_PYTHON)
 
   install(
     DIRECTORY ${CMAKE_SOURCE_DIR}/scripts
-    DESTINATION ${TARGETDIR_VER}
+    DESTINATION ${CMAKE_INSTALL_PREFIX}/share/blender/
     PATTERN ".git" EXCLUDE
     PATTERN ".gitignore" EXCLUDE
     PATTERN ".gitea" EXCLUDE
