diff -urN blender-2.64/source/creator/CMakeLists.txt blender-2.64-patched/source/creator/CMakeLists.txt
--- blender-2.64/source/creator/CMakeLists.txt.localedir	2011-12-13 20:46:18.000000000 +0100
+++ blender-2.64-patched/source/creator/CMakeLists.txt	2011-12-15 02:18:48.127562839 +0100
@@ -272,6 +272,11 @@
 	install(
 		DIRECTORY
 			${CMAKE_SOURCE_DIR}/release/datafiles/locale
+		DESTINATION share
+		PATTERN ".svn" EXCLUDE
+	)
+	install(
+		DIRECTORY
 			${CMAKE_SOURCE_DIR}/release/datafiles/fonts
 		DESTINATION ${TARGETDIR_VER}/datafiles
 		PATTERN ".svn" EXCLUDE
diff -urN blender-2.64/source/blender/blenfont/intern/blf_lang.c blender-2.64-patched/source/blender/blenfont/intern/blf_lang.c
--- blender-2.64/source/blender/blenfont/intern/blf_lang.c.localedir	2012-10-03 22:43:53.000000000 +0800
+++ blender-2.64-patched/source/blender/blenfont/intern/blf_lang.c	2012-10-07 09:04:54.524160065 +0800
@@ -113,7 +113,7 @@
 
 void BLF_lang_init(void)
 {
-	char *messagepath = BLI_get_folder(BLENDER_DATAFILES, "locale");
+	char *messagepath = "/usr/share/locale";
 
 	BLI_strncpy(global_encoding_name, SYSTEM_ENCODING_DEFAULT, sizeof(global_encoding_name));
 
