diff -urN blender-2.64/source/gameengine/Expressions/InputParser.cpp blender-2.64-patched/source/gameengine/Expressions/InputParser.cpp
--- blender-2.64/source/gameengine/Expressions/InputParser.cpp	2012-10-04 01:40:33.000000000 +1100
+++ blender-2.64-patched/source/gameengine/Expressions/InputParser.cpp	2012-10-05 19:20:54.294182152 +1100
@@ -382,7 +382,7 @@
 	if (s == sym) NextSym();
 	else {
 		STR_String msg;
-		msg.Format("Warning: " + Symbol2Str(s) + " expected\ncontinuing without it");
+		msg.Format("Warning: %s expected\ncontinuing without it", (const char *)Symbol2Str(s));
 
 //		AfxMessageBox(msg,MB_ICONERROR);
 
diff -urN blender-2.64/source/gameengine/Ketsji/KX_KetsjiEngine.cpp blender-2.64-patched/source/gameengine/Ketsji/KX_KetsjiEngine.cpp
--- blender-2.64/source/gameengine/Ketsji/KX_KetsjiEngine.cpp	2012-10-04 01:41:10.000000000 +1100
+++ blender-2.64-patched/source/gameengine/Ketsji/KX_KetsjiEngine.cpp	2012-10-05 21:01:09.165118457 +1100
@@ -1472,7 +1472,7 @@
 	{
 		for (int j = tc_first; j < tc_numCategories; j++)
 		{
-			debugtxt.Format(m_profileLabels[j]);
+			debugtxt.Format("%s", m_profileLabels[j]);
 			m_rendertools->RenderText2D(RAS_IRenderTools::RAS_TEXT_PADDED,
 			                            debugtxt.Ptr(),
 			                            xcoord,ycoord,
