--- intern/cycles/bvh/CMakeLists.txt.orig	2012-12-11 03:15:08.000000000 +0800
+++ intern/cycles/bvh/CMakeLists.txt	2012-12-12 20:11:15.134888709 +0800
@@ -33,4 +33,4 @@
 include_directories(${INC})
 include_directories(SYSTEM ${INC_SYS})
 
-add_library(cycles_bvh ${SRC} ${SRC_HEADERS})
+add_library(cycles_bvh STATIC ${SRC} ${SRC_HEADERS})
--- intern/cycles/device/CMakeLists.txt.orig	2012-12-11 03:15:08.000000000 +0800
+++ intern/cycles/device/CMakeLists.txt	2012-12-12 20:11:15.134888709 +0800
@@ -36,4 +36,4 @@
 include_directories(${INC})
 include_directories(SYSTEM ${INC_SYS})
 
-add_library(cycles_device ${SRC} ${SRC_HEADERS})
+add_library(cycles_device STATIC ${SRC} ${SRC_HEADERS})
--- intern/cycles/kernel/CMakeLists.txt.orig	2012-12-11 03:16:28.000000000 +0800
+++ intern/cycles/kernel/CMakeLists.txt	2012-12-12 20:11:46.665676012 +0800
@@ -143,7 +143,7 @@
 include_directories(${INC})
 include_directories(SYSTEM ${INC_SYS})
 
-add_library(cycles_kernel ${SRC} ${SRC_HEADERS} ${SRC_CLOSURE_HEADERS} ${SRC_SVM_HEADERS})
+add_library(cycles_kernel STATIC ${SRC} ${SRC_HEADERS} ${SRC_CLOSURE_HEADERS} ${SRC_SVM_HEADERS})
 
 if(WITH_CYCLES_OPTIMIZED_KERNEL)
 	set_source_files_properties(kernel_optimized.cpp PROPERTIES COMPILE_FLAGS "${CYCLES_OPTIMIZED_KERNEL_FLAGS}")
--- intern/cycles/kernel/osl/CMakeLists.txt.orig	2012-12-11 03:16:28.000000000 +0800
+++ intern/cycles/kernel/osl/CMakeLists.txt	2012-12-12 20:11:59.445995210 +0800
@@ -33,5 +33,5 @@
 include_directories(${INC})
 include_directories(SYSTEM ${INC_SYS})
 
-add_library(cycles_kernel_osl ${SRC} ${HEADER_SRC})
+add_library(cycles_kernel_osl STATIC ${SRC} ${HEADER_SRC})
 
--- intern/cycles/render/CMakeLists.txt.orig	2012-12-11 03:16:19.000000000 +0800
+++ intern/cycles/render/CMakeLists.txt	2012-12-12 20:11:15.144888959 +0800
@@ -69,5 +69,5 @@
 include_directories(${INC})
 include_directories(SYSTEM ${INC_SYS})
 
-add_library(cycles_render ${SRC} ${SRC_HEADERS})
+add_library(cycles_render STATIC ${SRC} ${SRC_HEADERS})
 
--- intern/cycles/subd/CMakeLists.txt.orig	2012-12-11 03:16:30.000000000 +0800
+++ intern/cycles/subd/CMakeLists.txt	2012-12-12 20:11:15.144888959 +0800
@@ -37,4 +37,4 @@
 include_directories(${INC})
 include_directories(SYSTEM ${INC_SYS})
 
-add_library(cycles_subd ${SRC} ${SRC_HEADERS})
+add_library(cycles_subd STATIC ${SRC} ${SRC_HEADERS})
--- intern/cycles/util/CMakeLists.txt.orig	2012-12-11 03:16:33.000000000 +0800
+++ intern/cycles/util/CMakeLists.txt	2012-12-12 20:11:15.144888959 +0800
@@ -70,4 +70,4 @@
 include_directories(${INC})
 include_directories(SYSTEM ${INC_SYS})
 
-add_library(cycles_util ${SRC} ${SRC_HEADERS})
+add_library(cycles_util STATIC ${SRC} ${SRC_HEADERS})
