diff -uraN blender-3.4.0/build_files/cmake/macros.cmake omv-blender-3.4.0/build_files/cmake/macros.cmake
--- blender-3.4.0/build_files/cmake/macros.cmake	2022-11-16 01:31:32.000000000 +0100
+++ omv-blender-3.4.0/build_files/cmake/macros.cmake	2022-12-08 17:34:46.083558435 +0100
@@ -288,7 +288,7 @@
 
   # message(STATUS "Configuring library ${name}")
 
-  add_library(${name} ${sources})
+  add_library(${name} STATIC ${sources})
 
   blender_target_include_dirs(${name} ${includes})
   blender_target_include_dirs_sys(${name} ${includes_sys})
