diff -urN blender-2.66a/source/blender/blenfont/intern/blf_lang.c blender-2.66a-patched/source/blender/blenfont/intern/blf_lang.c
--- blender-2.66a/source/blender/blenfont/intern/blf_lang.c	2013-01-21 21:46:01.000000000 +1100
+++ blender-2.66a-patched/source/blender/blenfont/intern/blf_lang.c	2013-03-15 21:03:53.855469743 +1100
@@ -78,15 +78,13 @@
 
 static void fill_locales(void)
 {
-	char *languages_path = BLI_get_folder(BLENDER_DATAFILES, "locale");
 	LinkNode *lines = NULL, *line;
 	char *str;
 	int idx = 0;
 
 	free_locales();
 
-	BLI_join_dirfile(languages_path, FILE_MAX, languages_path, "languages");
-	line = lines = BLI_file_read_as_lines(languages_path);
+	line = lines = BLI_file_read_as_lines("/usr/share/blender/locale/languages");
 
 	/* This whole "parsing" code is a bit weak, in that it expects strictly formated input file...
 	 * Should not be a problem, though, as this file is script-generated! */
@@ -185,7 +183,7 @@
 void BLF_lang_init(void)
 {
 #ifdef WITH_INTERNATIONAL
-	char *messagepath = BLI_get_folder(BLENDER_DATAFILES, "locale");
+	char *messagepath = "/usr/share/locale";
 
 	if (messagepath) {
 		bl_locale_init(messagepath, TEXT_DOMAIN_NAME);
diff -urN blender-2.66a/source/creator/CMakeLists.txt blender-2.66a-patched/source/creator/CMakeLists.txt
--- blender-2.66a/source/creator/CMakeLists.txt	2013-02-20 23:07:27.000000000 +1100
+++ blender-2.66a-patched/source/creator/CMakeLists.txt	2013-03-15 21:02:30.500323568 +1100
@@ -295,6 +295,16 @@
 	install(
 		DIRECTORY
 			${CMAKE_SOURCE_DIR}/release/datafiles/locale
+		DESTINATION share
+		PATTERN ".svn" EXCLUDE
+		PATTERN "languages" EXCLUDE
+	)
+	install(FILES
+			${CMAKE_SOURCE_DIR}/release/datafiles/locale/languages
+		DESTINATION share/blender/locale
+	)
+	install(
+		DIRECTORY
 			${CMAKE_SOURCE_DIR}/release/datafiles/fonts
 		DESTINATION ${TARGETDIR_VER}/datafiles
 		PATTERN ".svn" EXCLUDE
