diff --git a/build_files/cmake/Modules/FindOpenJPEG.cmake b/build_files/cmake/Modules/FindOpenJPEG.cmake
index d765103..f9d6998 100644
--- a/build_files/cmake/Modules/FindOpenJPEG.cmake
+++ b/build_files/cmake/Modules/FindOpenJPEG.cmake
@@ -42,12 +42,12 @@ FIND_PATH(OPENJPEG_INCLUDE_DIR
     ${_openjpeg_SEARCH_DIRS}
   PATH_SUFFIXES
     include
-    include/openjpeg-1.5
+    include/openjpeg-2.2
 )
 
 FIND_LIBRARY(OPENJPEG_LIBRARY
   NAMES
-    openjpeg
+    openjp2
   HINTS
     ${_openjpeg_SEARCH_DIRS}
   PATH_SUFFIXES
